test/ibmq/test_experiment.py

```diff
@@ -753,3 +773,30 @@ def _create_analysis_result(self):
         self.provider.experiment.upload_analysis_result(new_result)
         self.results_to_delete.append(new_result.uuid)
         return new_result
+
+    @classmethod
+    def _create_initial_test_data(cls):
+        """Creates an initial push of test data."""
+        # Generate 3 pieces of data.
+        for _ in range(3):
+            # Create an experiment
+            exp = Experiment(  # pylint: disable=unexpected-keyword-arg
+                provider=cls.provider,
+                backend_name=cls.BACKEND_NAME,
+                experiment_type='test',
+                tags=['qiskit-test', 'qiskit-test-1'],  # 2 tags for multi-tag checks
```

Comment:
```suggestion
        for x in range(3):
            # Create an experiment
            exp = Experiment(  # pylint: disable=unexpected-keyword-arg
                provider=cls.provider,
                backend_name=cls.BACKEND_NAME,
                experiment_type='test',
                tags=['qiskit-test', f'qiskit-test-{x}'],  # 2 tags for multi-tag checks
```

Nit but makes the experiments a bit less homogeneous.

Reply:
These are designed to be persistent test records so they would need to keep the `qiskit-test` tag exactly to show up in the test queries.